executor/workflows/tasks.py

```diff
@@ -254,16 +259,66 @@ def test_workflow_notification(account_id, workflow, message_type):
             for step, all_task_results in all_step_executions.items():
                 for result in all_task_results:
                     playbook_execution_log = PlaybookExecutionLog(
-                        playbook=playbook,
-                        playbook_step=step,
-                        playbook_task_definition=result['task'].proto,
-                        playbook_task_result=result['task_result_proto'],
+                        playbook=p_proto,
+                        step=step.proto,
+                        task=result['task'].proto,
```

Comment:
On line 254 we are deliberately converting the proto to dict while we are doing the reverse here. Change the code on line 254 to not convert the task into a dict

Reply:
On line 247 do we care about the ordering?

Reply:
Removed ordering, removed redundant proto <-> dict conversions